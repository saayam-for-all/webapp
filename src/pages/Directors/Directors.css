.person-card {
  transition: transform 0.2s ease;
  padding: 25px;
}

.person-card:hover {
  transform: scale(1.1);
  border-radius: 25%;
  box-shadow:
    10px 10px 10px 5px rgba(0, 0, 0, 0.2),
    -10px -6px 20px 0 rgba(0, 0, 0, 0.19);
  background-color: rgba(245, 245, 251, 0.808);
}

/* Style for the details display on hover */
.person-details {
  display: none;
  background-color: rgb(6, 5, 5);
  color: #fff;
  padding: 15px;
  position: absolute;
  bottom: 0;
  left: 2%;
  right: 0;
  transition: opacity 0.3s ease;
  border-radius: 5%;
}

.person-card:hover .person-details {
  display: block;
}

.person-card:hover .person_name {
  opacity: 0;
}

.person-card:hover .person_image {
  opacity: 0.5;
}

.container {
  align-items: center;
}

.Director_LinkedIn a {
  color: aqua;
}

.person_name {
  font-size: x-large;
}

.person_image {
  width: 100%;
  height: 250px;
  border-radius: 25%;
  box-shadow:
    0 4px 8px 0 rgba(0, 0, 0, 0.2),
    0 6px 20px 0 rgba(0, 0, 0, 0.19);
}

.person-details {
  width: 94%;
}

.bod_heading {
  color: rgb(11, 113, 255);
  margin-bottom: 2rem;
}

.person-card {
  margin-bottom: 3rem;
  width: 25%;
  float: left;
}

hr {
  width: 70%;
  margin-left: 15%;
}

/* @media only screen and (min-width: 820px) {
    .col-4 {
        width: 25%;
        float: left;
    }
} */

@media only screen and (min-width: 600px) and (max-width: 1024px) {
  .person-card {
    margin-bottom: 3rem;
    width: 50%;
    float: left;
  }
}

@media only screen and (max-width: 600px) {
  .person_image {
    width: 75%;
  }

  .person-details {
    display: none;
    background-color: rgba(0, 0, 0, 0.7);
    color: #fff;
    padding: 15px;
    position: absolute;
    bottom: 0;
    left: 10%;
    right: 0;
    transition: opacity 0.3s ease;
    width: 80%;
  }

  .person-card {
    margin-bottom: 3rem;
    width: 95%;
    float: none;
  }
}

.bod_container {
  margin-top: 1rem;
}
