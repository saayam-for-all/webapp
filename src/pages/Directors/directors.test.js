import { render } from "@testing-library/react";
import Directors from "./Directors";

describe("Directors", () => {
  it("renders correctly", () => {
    const tree = render(<Directors />);
    expect(tree).toMatchSnapshot();
  });
});
