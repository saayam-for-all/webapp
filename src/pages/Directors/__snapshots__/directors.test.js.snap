// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Directors renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="py-5"
      >
        <section>
          <h1
            class="text-center text-3xl font-bold text-black"
          >
            mockTranslate(DIRECTORS)
          </h1>
          <p
            class="text-center text-lg text-gray-600 my-4"
          >
            mockTranslate(DIRECTORS_DESCRIPTION)
          </p>
          <div
            class="directors flex items-center justify-center flex-wrap gap-10 my-20"
          >
            <div
              class=""
            >
              <div>
                <img
                  alt="Ramesh Maturu"
                  class="rounded-2xl shadow-lg"
                  height="400"
                  src=""
                  width="300"
                />
              </div>
              <div
                class="text-left self-start w-full px-6"
              >
                <h2
                  class="flex gap-2 mt-5"
                >
                  <a
                    class="text-black font-bold text-xl"
                    href="https://www.linkedin.com/in/rameshmaturu/"
                    target="_blank"
                  >
                    Ramesh Maturu
                  </a>
                </h2>
                <h3
                  class="flex gap-2"
                >
                  Director
                </h3>
              </div>
            </div>
            <div
              class=""
            >
              <div>
                <img
                  alt="Rao Panuganti"
                  class="rounded-2xl shadow-lg"
                  height="400"
                  src=""
                  width="300"
                />
              </div>
              <div
                class="text-left self-start w-full px-6"
              >
                <h2
                  class="flex gap-2 mt-5"
                >
                  <a
                    class="text-black font-bold text-xl"
                    href="https://www.linkedin.com/in/rao-panuganti-654443/"
                    target="_blank"
                  >
                    Rao Panuganti
                  </a>
                </h2>
                <h3
                  class="flex gap-2"
                >
                  Director
                </h3>
              </div>
            </div>
            <div
              class=""
            >
              <div>
                <img
                  alt="Naveen Sharma"
                  class="rounded-2xl shadow-lg"
                  height="400"
                  src=""
                  width="300"
                />
              </div>
              <div
                class="text-left self-start w-full px-6"
              >
                <h2
                  class="flex gap-2 mt-5"
                >
                  <a
                    class="text-black font-bold text-xl"
                    href="https://www.linkedin.com/in/nsharma2/"
                    target="_blank"
                  >
                    Naveen Sharma
                  </a>
                </h2>
                <h3
                  class="flex gap-2"
                >
                  Director
                </h3>
              </div>
            </div>
            <div
              class=""
            >
              <div>
                <img
                  alt="Rajeshwary Jaldu"
                  class="rounded-2xl shadow-lg"
                  height="400"
                  src=""
                  width="300"
                />
              </div>
              <div
                class="text-left self-start w-full px-6"
              >
                <h2
                  class="flex gap-2 mt-5"
                >
                  <a
                    class="text-black font-bold text-xl"
                    href="https://www.linkedin.com/in/rajeshwary-jaldu/"
                    target="_blank"
                  >
                    Rajeshwary Jaldu
                  </a>
                </h2>
                <h3
                  class="flex gap-2"
                >
                  Director & Treasurer
                </h3>
              </div>
            </div>
          </div>
        </section>
        <section>
          <div
            class="bg-gray-100 py-16 px-24"
          >
            <div
              class="text-center max-w-3xl mx-auto mb-12"
            >
              <h2
                class="font-extrabold text-3xl mb-4"
              >
                mockTranslate(EXECUTIVE_TEAM)
              </h2>
              <p
                class="text-gray-600 text-base leading-[0.5]"
              >
                mockTranslate(EXECUTIVE_DESCRIPTION_1)
              </p>
              <p
                class="text-gray-600 text-base leading-[0.5]"
              >
                mockTranslate(EXECUTIVE_DESCRIPTION_2)
              </p>
            </div>
            <div
              class="flex flex-wrap justify-center gap-10"
            >
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Rao K Bhethanabotla"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/42afdfb974265d5ede7e4b08aeff12e6a278dbec"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Rao K Bhethanabotla
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  Founder, CEO, CTO, President
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Rishita Rapuri"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/cd09abc1a5a391a8775f93bf52dc1c7bd3bde48c"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Rishita Rapuri
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  Human Resources Manager
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Rashmi"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/42134acaa608afdb1f30b2328f3ef05c63c7aee1"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Rashmi
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  Lead Application Developer
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Sriram Mahankali"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/8f181836a0df98180bc82bea522f96509eea4a8d"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Sriram Mahankali
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  Design and Creative Lead
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Shubam"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/f1155d03bc35b0497b2368237a95f6943836844a"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Shubam
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  AI/ML Team Lead
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Aatish Kayyath"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/e2fa182bfa332d6844de8112c9d522f03b2c678f"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Aatish Kayyath
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  ------
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Divya Tanwar"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/ce8a2aa126df5bac3884b96f7fe05a41fba85b38"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Divya Tanwar
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  -----
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Kavyasree"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/e1f54ababf3e0deebac0fef1e47e682a9823e34e"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Kavyasree
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  Lead Project Manager
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Purva Patil"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/6b58758ef77efe32c301deac6f4f0c1f6abb8fce"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Purva Patil
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  Lead Product Manager
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Mohana Krishanan R."
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/92f637ee67b17eef71df92364e32211645c6e3d0"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Mohana Krishanan R.
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  -----
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Rishab Ghosh"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/f0e403a7bcfa9d9be2f1c66f6fac4ca2218935ea"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Rishab Ghosh
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  -----
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Sugandha Agarwal"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/bcb73fdde6f2bca35c1db86c6ecb36f58fcd4fae"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Sugandha Agarwal
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  -----
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Uma"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/b6c938fe9f544ef0dacab7bd69eaca9a1a890fdb"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Uma
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  -----
                </div>
              </div>
              <div
                class="w-[180px] text-center"
              >
                <div
                  class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
                >
                  <img
                    alt="Meghana Karri"
                    class="w-full h-full object-cover"
                    src="https://cdn.builder.io/api/v1/image/assets/TEMP/091aca13a6e830ee2374f73b8d227506b2a90566"
                  />
                </div>
                <div
                  class="font-semibold text-base mt-3 text-left pl-1"
                >
                  Meghana Karri
                </div>
                <div
                  class="text-sm text-gray-500 mt-1 text-left pl-1"
                >
                  UX Lead
                </div>
              </div>
            </div>
          </div>
        </section>
        <section
          class="py-20"
        >
          <h1
            class="text-center text-3xl font-bold text-black"
          >
            mockTranslate(ADVISORS)
          </h1>
          <p
            class="text-center text-lg text-gray-600 my-4"
          >
            mockTranslate(ADVISORS_DESCRIPTION)
          </p>
          <div
            class="flex items-center justify-center flex-wrap gap-10 my-20"
          >
            <div
              class=""
            >
              <img
                alt="Madhukar Govindaraju"
                class="object-cover aspect-[2.5/3] rounded-xl"
                src=""
                width="250"
              />
              <div
                class="text-left self-start"
              >
                <h2
                  class="flex gap-2 mt-5"
                >
                  <a
                    class="text-black font-bold text-xl"
                    href="https://www.linkedin.com/in/madhukar/"
                    target="_blank"
                  >
                    Madhukar Govindaraju
                  </a>
                </h2>
                <h3
                  class="flex gap-2"
                >
                  Advisor
                </h3>
              </div>
            </div>
            <div
              class=""
            >
              <img
                alt="Sateesh Mucharla"
                class="object-cover aspect-[2.5/3] rounded-xl"
                src=""
                width="250"
              />
              <div
                class="text-left self-start"
              >
                <h2
                  class="flex gap-2 mt-5"
                >
                  <a
                    class="text-black font-bold text-xl"
                    href="https://www.linkedin.com/in/mucharla"
                    target="_blank"
                  >
                    Sateesh Mucharla
                  </a>
                </h2>
                <h3
                  class="flex gap-2"
                >
                  Advisor
                </h3>
              </div>
            </div>
          </div>
          <div
            class="flex flex-col items-center"
          >
            <h1
              class="text-center text-3xl font-bold text-black"
            >
              mockTranslate(Looking to join us?)
            </h1>
            <p
              class="text-center text-lg text-gray-600 my-4"
            >
              mockTranslate(Chat with our community and get in touch with our team!)
            </p>
            <button
              class="text-center bg-blue-500 hover:bg-blue-700 text-white px-4 py-2 my-2 rounded-2xl"
            >
              Join the community
            </button>
          </div>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="py-5"
    >
      <section>
        <h1
          class="text-center text-3xl font-bold text-black"
        >
          mockTranslate(DIRECTORS)
        </h1>
        <p
          class="text-center text-lg text-gray-600 my-4"
        >
          mockTranslate(DIRECTORS_DESCRIPTION)
        </p>
        <div
          class="directors flex items-center justify-center flex-wrap gap-10 my-20"
        >
          <div
            class=""
          >
            <div>
              <img
                alt="Ramesh Maturu"
                class="rounded-2xl shadow-lg"
                height="400"
                src=""
                width="300"
              />
            </div>
            <div
              class="text-left self-start w-full px-6"
            >
              <h2
                class="flex gap-2 mt-5"
              >
                <a
                  class="text-black font-bold text-xl"
                  href="https://www.linkedin.com/in/rameshmaturu/"
                  target="_blank"
                >
                  Ramesh Maturu
                </a>
              </h2>
              <h3
                class="flex gap-2"
              >
                Director
              </h3>
            </div>
          </div>
          <div
            class=""
          >
            <div>
              <img
                alt="Rao Panuganti"
                class="rounded-2xl shadow-lg"
                height="400"
                src=""
                width="300"
              />
            </div>
            <div
              class="text-left self-start w-full px-6"
            >
              <h2
                class="flex gap-2 mt-5"
              >
                <a
                  class="text-black font-bold text-xl"
                  href="https://www.linkedin.com/in/rao-panuganti-654443/"
                  target="_blank"
                >
                  Rao Panuganti
                </a>
              </h2>
              <h3
                class="flex gap-2"
              >
                Director
              </h3>
            </div>
          </div>
          <div
            class=""
          >
            <div>
              <img
                alt="Naveen Sharma"
                class="rounded-2xl shadow-lg"
                height="400"
                src=""
                width="300"
              />
            </div>
            <div
              class="text-left self-start w-full px-6"
            >
              <h2
                class="flex gap-2 mt-5"
              >
                <a
                  class="text-black font-bold text-xl"
                  href="https://www.linkedin.com/in/nsharma2/"
                  target="_blank"
                >
                  Naveen Sharma
                </a>
              </h2>
              <h3
                class="flex gap-2"
              >
                Director
              </h3>
            </div>
          </div>
          <div
            class=""
          >
            <div>
              <img
                alt="Rajeshwary Jaldu"
                class="rounded-2xl shadow-lg"
                height="400"
                src=""
                width="300"
              />
            </div>
            <div
              class="text-left self-start w-full px-6"
            >
              <h2
                class="flex gap-2 mt-5"
              >
                <a
                  class="text-black font-bold text-xl"
                  href="https://www.linkedin.com/in/rajeshwary-jaldu/"
                  target="_blank"
                >
                  Rajeshwary Jaldu
                </a>
              </h2>
              <h3
                class="flex gap-2"
              >
                Director & Treasurer
              </h3>
            </div>
          </div>
        </div>
      </section>
      <section>
        <div
          class="bg-gray-100 py-16 px-24"
        >
          <div
            class="text-center max-w-3xl mx-auto mb-12"
          >
            <h2
              class="font-extrabold text-3xl mb-4"
            >
              mockTranslate(EXECUTIVE_TEAM)
            </h2>
            <p
              class="text-gray-600 text-base leading-[0.5]"
            >
              mockTranslate(EXECUTIVE_DESCRIPTION_1)
            </p>
            <p
              class="text-gray-600 text-base leading-[0.5]"
            >
              mockTranslate(EXECUTIVE_DESCRIPTION_2)
            </p>
          </div>
          <div
            class="flex flex-wrap justify-center gap-10"
          >
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Rao K Bhethanabotla"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/42afdfb974265d5ede7e4b08aeff12e6a278dbec"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Rao K Bhethanabotla
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                Founder, CEO, CTO, President
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Rishita Rapuri"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/cd09abc1a5a391a8775f93bf52dc1c7bd3bde48c"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Rishita Rapuri
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                Human Resources Manager
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Rashmi"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/42134acaa608afdb1f30b2328f3ef05c63c7aee1"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Rashmi
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                Lead Application Developer
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Sriram Mahankali"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/8f181836a0df98180bc82bea522f96509eea4a8d"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Sriram Mahankali
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                Design and Creative Lead
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Shubam"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/f1155d03bc35b0497b2368237a95f6943836844a"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Shubam
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                AI/ML Team Lead
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Aatish Kayyath"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/e2fa182bfa332d6844de8112c9d522f03b2c678f"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Aatish Kayyath
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                ------
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Divya Tanwar"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/ce8a2aa126df5bac3884b96f7fe05a41fba85b38"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Divya Tanwar
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                -----
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Kavyasree"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/e1f54ababf3e0deebac0fef1e47e682a9823e34e"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Kavyasree
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                Lead Project Manager
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Purva Patil"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/6b58758ef77efe32c301deac6f4f0c1f6abb8fce"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Purva Patil
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                Lead Product Manager
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Mohana Krishanan R."
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/92f637ee67b17eef71df92364e32211645c6e3d0"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Mohana Krishanan R.
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                -----
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Rishab Ghosh"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/f0e403a7bcfa9d9be2f1c66f6fac4ca2218935ea"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Rishab Ghosh
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                -----
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Sugandha Agarwal"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/bcb73fdde6f2bca35c1db86c6ecb36f58fcd4fae"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Sugandha Agarwal
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                -----
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Uma"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/b6c938fe9f544ef0dacab7bd69eaca9a1a890fdb"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Uma
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                -----
              </div>
            </div>
            <div
              class="w-[180px] text-center"
            >
              <div
                class="w-[200px] h-[240px] overflow-hidden rounded-lg mx-auto"
              >
                <img
                  alt="Meghana Karri"
                  class="w-full h-full object-cover"
                  src="https://cdn.builder.io/api/v1/image/assets/TEMP/091aca13a6e830ee2374f73b8d227506b2a90566"
                />
              </div>
              <div
                class="font-semibold text-base mt-3 text-left pl-1"
              >
                Meghana Karri
              </div>
              <div
                class="text-sm text-gray-500 mt-1 text-left pl-1"
              >
                UX Lead
              </div>
            </div>
          </div>
        </div>
      </section>
      <section
        class="py-20"
      >
        <h1
          class="text-center text-3xl font-bold text-black"
        >
          mockTranslate(ADVISORS)
        </h1>
        <p
          class="text-center text-lg text-gray-600 my-4"
        >
          mockTranslate(ADVISORS_DESCRIPTION)
        </p>
        <div
          class="flex items-center justify-center flex-wrap gap-10 my-20"
        >
          <div
            class=""
          >
            <img
              alt="Madhukar Govindaraju"
              class="object-cover aspect-[2.5/3] rounded-xl"
              src=""
              width="250"
            />
            <div
              class="text-left self-start"
            >
              <h2
                class="flex gap-2 mt-5"
              >
                <a
                  class="text-black font-bold text-xl"
                  href="https://www.linkedin.com/in/madhukar/"
                  target="_blank"
                >
                  Madhukar Govindaraju
                </a>
              </h2>
              <h3
                class="flex gap-2"
              >
                Advisor
              </h3>
            </div>
          </div>
          <div
            class=""
          >
            <img
              alt="Sateesh Mucharla"
              class="object-cover aspect-[2.5/3] rounded-xl"
              src=""
              width="250"
            />
            <div
              class="text-left self-start"
            >
              <h2
                class="flex gap-2 mt-5"
              >
                <a
                  class="text-black font-bold text-xl"
                  href="https://www.linkedin.com/in/mucharla"
                  target="_blank"
                >
                  Sateesh Mucharla
                </a>
              </h2>
              <h3
                class="flex gap-2"
              >
                Advisor
              </h3>
            </div>
          </div>
        </div>
        <div
          class="flex flex-col items-center"
        >
          <h1
            class="text-center text-3xl font-bold text-black"
          >
            mockTranslate(Looking to join us?)
          </h1>
          <p
            class="text-center text-lg text-gray-600 my-4"
          >
            mockTranslate(Chat with our community and get in touch with our team!)
          </p>
          <button
            class="text-center bg-blue-500 hover:bg-blue-700 text-white px-4 py-2 my-2 rounded-2xl"
          >
            Join the community
          </button>
        </div>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
